File: app/src/main/java/com/example/gaijin/countriestocities/Country.java
package com.example.gaijin.countriestocities;

import io.realm.RealmList;
import io.realm.RealmObject;
import io.realm.annotations.PrimaryKey;

public class Country extends RealmObject {

    @PrimaryKey
    private String countryName;
    private RealmList<City> cities;

    public Country() {
    }

    public Country(String countryName, RealmList<City> cities) {
        this.countryName = countryName;
        this.cities = cities;
    }

    public String getCountryName() {
        return countryName;
    }

    public void setCountryName(String countryName) {
        this.countryName = countryName;
    }

    public RealmList<City> getCities() {
        return cities;
    }

    public void setCities(RealmList<City> cities) {
        this.cities = cities;
    }
}

File: app/src/main/java/com/example/gaijin/countriestocities/dataclasses/Country.kt
package com.example.gaijin.countriestocities.dataclasses

import com.google.gson.annotations.SerializedName


data class Country(

        @SerializedName("country_name")
        var name: String,

        @SerializedName("country_code")
        var code: String,

        @SerializedName("flag")
        var flage: String,

        @SerializedName("cities")
        var cities: List<String>
)